antismash/modules/nrps_pks/transat_ks_analysis/data/RAxML_bestTree.649KS_sequences_hmmalign_raxml_renamed.refpkg/RAxML_info.649KS_sequences_hmmalign_raxml.tre

```diff
@@ -0,0 +1,79 @@
+

```

Comment:
These data files have some pretty obnoxius paths. If it's long to include all the relevant info, that should be in a generation script included with the files so we can regenerate them consistently at some point in the future (and the pre-processed data, if it's small, maybe should be included, or if it's not small, have information on where to get it).